src/pages/Address/AddressComp.tsx

```diff
@@ -261,32 +193,70 @@ export const AddressOverviewCard: FC<{ address: Address }> = ({ address }) => {
 
       <CardCellsLayout type="leftSingle-right" cells={overviewItems} borderTop />
 
-      {udtAccounts.length || cotaList?.length ? (
+      {udts.length > 0 || (cotaList?.length && cotaList.length > 0) || inscriptions.length > 0 ? (
```

Comment:
What's the difference between `udts.length` and `udts.length > 0`

Reply:
> What's the difference between `udts.length` and `udts.length > 0`

No difference, I think maybe it is more semantical to read the code "the length is larger than 0"